.github/workflows/scheduled.yml

```diff
@@ -163,6 +163,12 @@ jobs:
       - name: Validate build
         run: ./script/validate-build.rb
 
+      - name: Sign API
+        env:
+          JWS_SIGNING_KEY_ID: homebrew-1
+          JWS_SIGNING_KEY: ${{ secrets.JWS_HOMEBREW_1 }}
```

Comment:
Is this setup yet or does it need set?

Reply:
I've added one I generated on my machine. I will destroy the copy on my machine but can put it in 1Password before I do that.

Reply:
Perfect, thanks and: yes please on 1Password. Please use the Ops vault (rather than Maintainers).